asserts/crypto.go

```diff
@@ -191,6 +191,8 @@ func decodeSignature(signature []byte) (Signature, error) {
 type PublicKey interface {
 	// Fingerprint returns the key fingerprint.
 	Fingerprint() string
+	// KeyId returns the key id used to match signatures to their signing key.
+	KeyID() string
```

Comment:
@niemeyer it's more coming from the PGP terminology, it almost never says just id for keys but key id or keyid, it uses just fingerprint though sometimes